internal/repositories/users.go

```diff
@@ -1,21 +1,45 @@
 package configs
 
 import (
+	"database/sql"
 	"errors"
 	"lets-go-chat/internal/models"
 )
 
-var userStorage = make(map[string]models.User)
+type DBConnection struct {
+	dbCon *sql.DB
+}
+
+var UserNotFound = errors.New("user not found")
+
+var UserWasNotSaved = errors.New("user was not saved in db")
 
-var UserNotFound = errors.New("USER NOT FOUND")
+var dbConnection DBConnection
 
-func SaveUser(user models.User) {
-	userStorage[user.UserName] = user
+func NewUsersRepository(db *sql.DB){
+	dbConnection = DBConnection{
```

Comment:
we should discuss it, but today we have lecture were you can get some  knowledge how to bind functions to structs